EasyPost/Client.cs

```diff
@@ -51,7 +52,7 @@ public Client(ClientConfiguration clientConfiguration, HttpClient? customHttpCli
             {
                 Assembly assembly = typeof(Client).Assembly;
                 FileVersionInfo info = FileVersionInfo.GetVersionInfo(assembly.Location);
-                _libraryVersion = info.FileVersion;
+                _libraryVersion = info.FileVersion ?? "Unknown";
```

Comment:
How did this sneak in or was it added? Rebase gone wrong somewhere? I thought this was already in the repo.

Reply:
I believe I added that in earlier, but it's just a fallback in case the .FileVersion function fails (returns null, but `_libraryVersion` can't have a null value assigned to it)